sites/public/lib/filterUrlLink.tsx

```diff
@@ -0,0 +1,15 @@
+import { encodeToFrontendFilterString, ListingFilterState } from "@bloom-housing/ui-components"
+
+export function getFilterUrlLink(eligibilityRequirements) {
+  /** This is where any logic for the filtered results from the eligibility questionnaire should be located.*/
+
+  const SENIOR_AGE = 62
+
+  const params: ListingFilterState = {}
+
+  if (eligibilityRequirements.age >= SENIOR_AGE) {
+    params.seniorHousing = true
+  }
```

Comment:
I think this is the opposite? Shouldn't it be < SENIOR_AGE, assign to false?

Reply:
Like the includeNull parameter, we only need the true state, so when the input age is greater than or equal to 62 we include it as true, otherwise its presumed false.   This was a carryover from Elen's, just made into an if statement.

Reply:
But it's the opposite in the place where you're removing it - see `income.tsx`